src/main/java/org/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -917,7 +918,8 @@ public void actionPerformed(ActionEvent e) {
         private AutoLinkAction() {
             putValue(Action.SMALL_ICON, IconTheme.JabRefIcon.AUTO_FILE_LINK.getIcon());
             putValue(Action.SHORT_DESCRIPTION,
-                    Localization.lang("Automatically set file links for this entry") + " (Alt-F)");
+                    Localization.lang("Automatically set file links for this entry") +
+                            " (" + KeyStrokeToString.keyStrokeToString(Globals.getKeyPrefs().getKey(KeyBinding.AUTOMATICALLY_LINK_FILES)) + ")");
```

Comment:
Why don't you call Globals.getKeyPrefs().get(KeyBinding.AUTOMATICALLY_LINK_FILES).get()=
The get Method returns already the binding name (or the default binding, e.g. ctrl+z as an Optional<String>  so no need to rely on the KeyStroke stuff

Reply:
Did not find the method.